projects/storefrontapp-e2e-cypress/cypress/e2e/regression/asm/create-customer.e2e.cy.ts

```diff
@@ -156,8 +73,7 @@ context('Assisted Service Module', () => {
       asm.asmCloseCreateCustomerDialog();
     });
 
-    // TODO(#CXSPA-5165): enable this case
-    it.skip('should be not able to create a new customer with duplicated user data by agent (CXSPA-1594)', () => {
+    it('should be not able to create a new customer with duplicated user data by agent (CXSPA-1594)', () => {
```

Comment:
The backend version of CI is [2211.1](https://sap-cx.slack.com/archives/CAC8PULNN/p1700255196274999?thread_ts=1700255152.375369&cid=CAC8PULNN)4, so enable this case